views/js/scoring/processor/expressions/operators/equal.js

```diff
@@ -0,0 +1,106 @@
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technlogies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+/**
+ * The equal operator processor.
+ * @see http://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element10654
+ *
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+define([
+    'lodash',
+    'taoQtiItem/scoring/processor/expressions/preprocessor',
+    'taoQtiItem/scoring/processor/errorHandler'
+], function(_, preProcessor, errorHandler){
+    'use strict';
+
+    /**
+     * Process operands and returns the equal.
+     * @type {OperatorProcessor}
+     * @exports taoQtiItem/scoring/processor/expressions/operators/equal
+     */
+    var equalProcessor = {
+
+        engines: {
+            exact: function (x, y) {
+                return x === y;
+            },
+            absolute: function (x, y, includeLowerBound, includeUpperBound, tolerance) {
+                var lower = includeLowerBound ? y >= x - tolerance[0] : y > x - tolerance[0],
+                    upper = includeUpperBound ? y <= x + tolerance[1] : y < x + tolerance[1];
+                return lower && upper;
+            },
+            relative: function (x, y, includeLowerBound, includeUpperBound, tolerance) {
+                var lower = includeLowerBound ? y >= x - (1 - tolerance[0] / 100) : y > x - (1 - tolerance[0] / 100),
+                    upper = includeUpperBound ? y <= x + (1 - tolerance[1] / 100) : y < x + (1 - tolerance[1] / 100);
+
+                return lower && upper;
+            }
+        },
+
+        constraints : {
+            minOperand : 2,
+            maxOperand : 2,
+            cardinality : ['single'],
+            baseType : ['integer', 'float']
+        },
+
+        operands   : [],
+
+        /**
+         * Process the equal of the operands.
+         * @returns {?ProcessingValue} is equal or null
+         */
+        process : function(){
+
+            var result = {
+                cardinality : 'single',
+                baseType : 'boolean'
+            };
+
+            var activeToleranceEngine = _.isFunction(this.expression.attributes.activeToleranceEngine) ? this.expression.attributes.activeToleranceEngine : equalProcessor.engines.exact,
+                tolerance = _.isArray(this.expression.attributes.tolerance) ? this.expression.attributes.tolerance : [],
```

Comment:
Parse the value using the preprocessor since it is a `floatOrVariableRef` (merge newrunner into this branch before)